CONTRIBUTING.md

```diff
@@ -151,6 +152,125 @@ Validate authentication using the username `alice` with the password `alice`:
+
+###### Load Google Cloud Authenticator
+
+Save the following policy into `author-gce.yml` file:
+
+```yaml
+- !policy
+  id: conjur/authn-gce
+  body:
+  - !webservice
+  
+  - !group
+    id: apps
+  
+  - !permit
+    role: !group apps
+    privilege: [ read, authenticate ]
+    resource: !webservice
+```
+
+Load the policy into conjur:
+
+`conjur policy load root authn-gce.yml`
+
+###### Define a Google Cloud Resource as a Host in Conjur
+
+The information obtained from Google Cloud will be defined as annotations in the Conjur host to create the correlation required by Conjur to validate the caller (Google Cloud Identity token contains this attributes as claims).
+
+Below is a mapping between Conjur host annotations and the ID token claims:
+
+| Host Annotation       | Claim                               | Description                                                  |
+| :-------------------- | :---------------------------------- | :----------------------------------------------------------- |
+| instance-name         | google/compute_engine/instance_name | The name of the instance to which this token belongs. [More info on compute engine.](https://cloud.google.com/compute) |
+| project-id            | google/compute_engine/project_id    | The ID for the project where you created the instance.       |
+| service-account-email | email                               | The service account's name is a unique identifier; it appears in the service account's email address that is provisioned during creation, such as `sa-name@project-id.iam.gserviceaccount.com`. [More info on service accounts](https://cloud.google.com/iam/docs/service-accounts). |
+| service-account-id    | sub                                 | The subject of the token, i.e.the unique ID for the service account that you associated with your instance. [More info on service accounts](https://cloud.google.com/iam/docs/service-accounts). |
+
+*Note: all annotations must start with authn-gce/ prefix*
+
+Your policy might look similar to the following example:
+
+```yaml
+- !policy
+ id: gce-apps
+ body:
+  - !group
+ 
+  - &hosts
+   - !host
+    id: testapp
+    annotations:
+     authn-gce/instance-name: vm-for-gcp
+     authn-gce/project-id: eng-serenity-2313
+     authn-gce/service-account-id: 1234567890987654321
+     authn-gce/service-account-email: 1234567-compute@developer.gserviceaccount.com
+     
+  - !grant
+   role: !group
+   members: *hosts
+     
+  - !grant
+   role: !group /conjur/authn-gce/apps
+   member: !group gce-apps
+```
+
+###### Authenticate using Google Cloud Authenticator
+
+Send authentication request to Conjur using the account `myorg` and the host resource `gce-apps/testapp` and together with Google Identity JWT token.
```

Comment:
I understand that token is already included in JWT, so you don;t say JWT token, just simply JWT
And a Google Identity token is a JWT.
 So we need to reword.